src/commands/default/env.ts

```diff
@@ -58,11 +59,12 @@ export default class Peers extends BaseCommand<typeof Peers> {
       newFluenceEnv === "custom" &&
       fluenceConfig.customFluenceEnv === undefined
     ) {
-      await ensureCustomRelays(fluenceConfig);
+      await ensureCustomNodes(fluenceConfig);
```

Comment:
I would avoid using name "Node"/"Nodes" to mean Nox instances. Better call them Peers or Relays.

Node is way too overloaded, and clashes with Node.js.

Reply:
agree. Maybe rename this `@fluencelabs/fluence-network-environment` package - that's entirely the reason I call it like this - for consistency

Reply:
I will call this type and these functions `AddrAndPeerId` cause that's what these object are actually

Reply:
I think `PeerAddress` is easier to grasp. `and` in names is hard.

Reply:
it's a bit harder but that's literally what we have here. Maybe we shouldn't have it at all? Always have just a simple array of multiaddrs and a simple function to get peerId from multiaddr? js-client users would rarely need peerId right? Maybe we expose only multiaddrs?

Reply:
js-client always needs peer id in the multiaddr. I guess that's why there's a separate peer_id field in there.

but yeah, just multiaddr would be much simpler. it just would take more validation.

Reply:
But the problem is that js-client accepts mutiadr as a string anyway and validates it. So there is no much reason to have peer_id field in there. It's totally derivable from address alone